File: BahamasSystem/BahamasSystem/OrderSizer.cpp
#include "OrderSizer.h"

OrderSizer::OrderSizer()
{
}

OrderSizer::~OrderSizer()
{
}

//Basic Order Sizing
void OrderSizer::SizeOrder(MarketOrder& order) {
	if (order.Units == 0)
		order.Units = 1;
}

File: BahamasEngine/order_sizer.py
class MarketOrder(object):

    def __init__(self, option_id, action, units, price):
        self.option_id = option_id
        self.action = action
        self.units = units
        self.price = price


def size_order(m_order, portfolio):

    #TODO Handle position sizing
    return m_order